ci/scripts/unpause-pipeline.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+target=$( cat ~/.flyrc | yq  ".targets | keys | .[0]")
```

Comment:
Simply taking the first target is a bit dangerous. Can you change this to select the `app-autoscaler-release` target via name explicitly?